File: RetrofitTest/app/src/main/java/com/example/jestina/retrofitex/view/MainActivity.java
package com.example.jestina.retrofitex.view;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.example.jestina.retrofitex.R;
import com.example.jestina.retrofitex.contract.MainContract;
import com.example.jestina.retrofitex.presenter.MainPresenter;

public class MainActivity extends AppCompatActivity implements MainContract.View {
    MainPresenter mPresenter;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        initView();
        initPresenter();
    }

    @Override
    public void initView() {
        setContentView(R.layout.activity_main);
    }

    @Override
    public void initPresenter() {
        mPresenter = new MainPresenter();
        mPresenter.initPresenter(this);
    }
}

File: CountUpAni_Kt/app/src/main/java/com/example/jestina/countupani_kt/MainActivity.kt
package com.example.jestina.countupani_kt

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import com.example.jestina.countupani_kt.utils.YWLog

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        YWLog.d("adsf")
    }
}
